echopype/convert/api.py

```diff
@@ -369,10 +382,23 @@ def open_raw(
 
     Notes
     -----
-    ``use_swap=True`` is only available for the following
+    In a case of a large memory footprint, the program will determine if using
+    a temporary swap space is needed. If so, it will use that space
+    during conversion to prevent out of memory errors.
+    Users can override this behaviour by either passing ``"swap"`` or ``"no_swap"``
+    into the ``destination_path`` argument.
+    This feature is only available for the following
     echosounders: EK60, ES70, EK80, ES80, EA640. Additionally, this feature
     is currently in beta.
```

Comment:
```suggestion
    When the required memory footprint is large, the program will determine if using
    a temporary swap space is needed during conversion to prevent out of memory errors.
    Users can override this behavior by setting ``destination_path="swap"``
     or ``destination_path="no_swap"``.
    This feature is only available for the following
    echosounders: EK60, ES70, EK80, ES80, EA640. 
    This is a beta feature.
```